### ISSUE TITLE
Add Apple

### ISSUE THREAD

**Issue by Madis0** [urbandroid-team/dont-kill-my-app#23]
Mainly just for the sake of proving what is even possible in iOS compared to Android, because after seeing this site, there will obviously be comparisons. 

Reference: https://developer.apple.com/library/archive/documentation/iPhone/Conceptual/iPhoneOSProgrammingGuide/BackgroundExecution/BackgroundExecution.html#//apple_ref/doc/uid/TP40007072-CH4-SW1

**Comment by Artaud** [comment]
The main difference is that Apple has it documented. What the developers on Android are fed up with is that we have to figure these things out on our own, the vendors don't care about us...

An Apple section might be pretty funny :) Would you like to create it? If you just posted the text I'll be happy to turn it into a section on the site

**Comment by yccheok** [comment]
I don't see a point of having Apple section, as Apple has a document on background process restriction, and Apple's products work as per document. The rule of game works as per it is. Nothing to complain about.

But not Android. Google has a document too on background process restriction, and Android's product DO NOT work as per document. The rule of game DOES NOT work as per it is. We need to complain :)

In short, Google should really hold the final control on background process restriction policy, not device OEM.
